services/QuillLMS/app/views/application/_premium_footer.html.erb

```diff
@@ -1,12 +1,7 @@
-<% if !current_user || (current_user.teacher? && !current_user.is_premium?) %>
-  <div class="premium-footer">
-    <div class="content-container">
-      <div class="text-section">
-        <h2>Learn More About Quill Premium</h2>
-        <p>Premium subscriptions for schools and districts interested in priority technical support, additional reporting, and support from Quill's professional learning team--plus an option for individual teachers</p>
-      </div>
+<script> const global = globalThis; </script>
```

Comment:
What is the purpose of this assignment?

Reply:
On the home page, public page, and session login pages, I was unable to load the React component and getting a console error "global is not defined". After digging around it seemed like this solution worked:

https://github.com/jpuri/react-draft-wysiwyg/issues/1317

Reply:
@cissyxyu It's unlikely that this directive is necessary, since no other pages require it (although see _head for the vite shim). Would you mind poking at it a bit more? If you don't have a solution by Wednesday EST, ping me, and we can look into it together.

Reply:
Agreed that this feels like something we should be able to get rid of.

Reply:
@happythenewsad I dug around a bit and it looks like I just need to include the same vite shim that _head already has. This is because _head and _twenty_seventeen_home (which loads the current home page plus connect, grammar, etc. overview pages) are currently two different loading paths. So I copied the vite shim and added it to twenty_seventeen_home. Do you think this would be an appropriate solution?

Reply:
Yeah, I think it's fine and necessary to put the shim into `_twenty_seventeen_home` home (although we should collapse this layout into `application.html.erb` in a different PR). 

However, note that the `activity.html.erb` layout includes the `_head` partial. `_head` has the shim, so `activity.html.erb` shouldn't need to add it a second time. 

I checked your branch, removed the shim from `activity.html.erb`, and loaded an Evidence activity. Looks fine:
<img width="1918" alt="Screenshot 2024-03-25 at 10 28 15 AM" src="https://github.com/empirical-org/Empirical-Core/assets/90669/f68a3bb4-1044-47cb-a469-be41ed9d37e3">